CONTRIBUTING.md

```diff
@@ -0,0 +1,147 @@
+```
+
+And activate it with
+
+```bash
+conda activate theme-env
+```
+
+With pip/venv, you can create a virtual environment with
+
+```bash
+python -m venv theme-env
+```
+
+And activate it with the following on Linux and macOS,
+
+```bash
+source theme-env/bin/activate
+```
+
+or on Windows (cmd),
+
+```cmd
+.\theme-env\Scripts\activate.bat
+```
+
+Regardless of the tool you use, make sure to remember to always activate your environment before using it.
+
+
+### Install the development dependencies
+
+To get the consistent pinned versions of the development dependencies, install the deps from the requirements file.
+If using conda, run:
+
+```bash
+conda install -c conda-forge --file docs/requirements.txt
+```
+
+Or, using pip,
+
+```bash
+python -m pip -r docs/requirements.txt
+```
+
+
+### Installing the theme package itself
+
+Finally, install the theme itself in editable mode, along with its ``npm`` dependencies (for rebuilding any CSS/JS files with Webpack; you'll need NodeJS/npm installed and accessable on your path):
+
+```bash
+pip install -e .
+npm install
+```
+
+
+
+## Building and testing the theme
+
+In the root of this repository, to run a continuously updating local webserver with the theme updated interactively, execute ``npm run build:dev`` to build a continually updating test site that will allow you to preview your changes as you make them.
+Once you're satisfied, you can use ``npm run build:production`` to build the theme package for use on the actual docs repo, and finally build the docs as normal (e.g. ``make html``) in the same environment to observe the changes using the updated theme.
+
+
+
+## Making Your Changes
+
+To start working on a new PR, you need to execute these commands, filling in the branch names where appropriate (``<FEATURE-BRANCH>`` is the branch you'll be creating to store your changes, e.g. ``fix-startup-bug`` or ``add-widget-support``):
+
+```bash
+git checkout develop_spyder
+git pull upstream develop_spyder
+git checkout -b <FEATURE-BRANCH>
+```
+
+Once you've made and tested your changes, commit them with a descriptive message of 74 characters or less written in the imperative tense, with a capitalized first letter and no period at the end (our pre-commit hooks will check that for you, so make sure to install them).
+For example:
+
+```bash
+git commit -am "Fix style issues on mobile browsers"
+```
+
+After making your changes, make sure to test them with ``npm run build:dev`` and/or ``npm run build:production`` followed by the docs build command (e.g. ``make html``).
+
+
+
+## Pushing your Changes and Submitting a Pull Request
+
+Now that your changes are ready to go, you'll need to push them to your fork.
+
+```bash
+git push -u origin <FEATURE-BRANCH>
+```
+
+Finally, create a pull request to the [spyder-ide/spyder-docs-sphinx-theme repository](https://github.com/spyder-ide/spyder-docs-sphinx-theme/) on Github.
+
+Thanks, and we look forward to your generous contributions!
```

Comment:
Maybe put instead something like:
```
Thank you for taking the time to read our contribution guide!

Let us know if you have any question and we look forward for any type of contribution!
```

Reply:
I worked the first sentence in at the end of the "Pushing/PR" section here, and added a revised version of the second to the end of the introductory paragraph so people know it up front and it is under a more appropriate heading.